statsmodels/stats/pscore_match.py

```diff
@@ -0,0 +1,242 @@
+'''Propensity Score Matchig
+
+
+Author: 
+License: BSD-3
+
```

Comment:
Can you add some notes here that provide some literature references for your implementation, and comments about how it relates to propensity score procedures in other packages like R and Stata?